src/api/client/AccountManagement.ts

```diff
@@ -276,4 +276,17 @@ export class AccountManagement {
 
     return new Subsidy({ beneficiary: beneficiaryAddress, subsidizer: subsidizerAddress }, context);
   }
+
+  /**
+   * Returns validation error if @param args.address is not a valid ss58 string for the connected network
+   */
+  public validateAddress(args: { address: string }): Error | null {
```

Comment:
Should it be an assertions or should it return a boolean? I have no clear answer.

Reply:
Something like isValid return true or false would be better IMO